docs/content/en/functions/hmac.md

```diff
@@ -0,0 +1,35 @@
+---
+title: hmac
+# linktitle: hmac
+description: Compute the cryptographic checksum of a message.
+godocref:
+date: 2020-05-29
+publishdate: 2020-05-29
+lastmod: 2020-05-29
+categories: [functions]
+menu:
+  docs:
+    parent: "functions"
+keywords: [hmac,checksum]
+signature: ["hmac HASH_TYPE MESSAGE KEY"]
+workson: []
+hugoversion:
+relatedfuncs: [hmac]
+deprecated: false
+aliases: [hmac]
+---
+
+`hmac` hashes the concatenation of a message and a secret key with the given hash function
+and returns its checksum.
+
+```
+{{ hmac "sha256" "Hello Gopher!" "Hello world, gophers!"}},
```

Comment:
Use a more obvious example key (such as `SecretKey!`).  Same goes for the other examples and the tests.